terraform/modules/monitoring/default_dashboard.json

```diff
@@ -0,0 +1,764 @@
+{
+    "annotations": {
+        "list": [
+            {
+                "builtIn": 1,
+                "datasource": "-- Grafana --",
+                "enable": true,
+                "hide": true,
+                "iconColor": "rgba(0, 211, 255, 1)",
+                "name": "Annotations & Alerts",
+                "target": {
+                    "limit": 100,
+                    "matchAny": false,
+                    "tags": [],
+                    "type": "dashboard"
+                },
+                "type": "dashboard"
+            }
+        ]
+    },
+    "description": "",
+    "editable": false,
+    "fiscalYearStartMonth": 0,
+    "graphTooltip": 0,
+    "links": [],
+    "liveNow": false,
+    "panels": [
+        {
+            "datasource": {
+                "type": "prometheus",
+                "uid": "P1809F7CD0C75ACF3"
```

Comment:
I'm not sure (I couldn't find any documentation on this field, or any other `uid` field, better than https://grafana.com/docs/grafana/latest/dashboards/json-model/ which leaves a lot to be desired); but it doesn't appear to need to match anything outside the dashboard definition -- at least, I generated this dashboard JSON in my `facilitator` environment, then deployed it to my `pha` environment, and it still worked there.

That said, I have no idea how we would match up to the correct data source other than this field. So I don't know _why_ it works, only that it does.

Reply:
OK. Well in that case, I think we can take this change and then see what happens in staging and prod. The worst case is that the dashboard won't work because the `uid` is invalid, which leaves us no worse off than we are now, and we can fix-forward from there.